helm/auto-oncall-chart/templates/certificate.yaml

```
@@ -0,0 +1,20 @@
+{{- if .Values.Installation.V1.Autooncall.TLS.enabled }}
```

```suggestion
{{- if .Values.Installation.V1.AutoOncall.TLS.enabled }}
```

I find this a bit easier to read.